src/dataset-builder/CohortEditor.test.ts

```diff
@@ -359,7 +354,7 @@ describe('CohortEditor', () => {
   it('renders criteria group', async () => {
     // Arrange
     const count = 12345;
-    mockGetCounts(count);
+    mockDataRepo(getSnapshotBuilderCountMock(count));
```

Comment:
Do all tests need to mock the count API now? If so, is there a way to share that, or can we only mock the entire datarepo API at once?

Reply:
I don't think all of them need it. I thought about doing a before each, but I wasn't sure how to append mocks so that it didn't override the mock when we needed new mocking